app/src/main/java/com/example/music_app/ui/theme/Sizes.kt

```diff
@@ -0,0 +1,7 @@
+package com.example.music_app.ui.theme
+
+import androidx.compose.ui.unit.dp
+
+val SHAPE_SIZE = 8.dp
+val BASE_PADDING = 10.dp
+val IMAGE_SIZE = 80.dp
```

Comment:
I would recommend defining dimensions somehow like this
```suggestion
@Immutable
data class Dimensions(
    val spacing00: Dp = 0.dp,
    val spacing04: Dp = 4.dp
    
    ...
```

Reply:
With this adjustment dimension could be easily applied to your app's theme and used all across the application. Same goes with the colors